src/extensions/default/InlineColorEditor/ColorEditor.js

```diff
@@ -215,12 +234,12 @@ define(function (require, exports, module) {
     /** Add event listeners to the format button bar */
     ColorEditor.prototype._bindColorFormatToRadioButton = function (buttonClass, propertyName, value) {
         var handler,
-            _this = this;
+            self = this;
         handler = function (event) {
-            var colorObject, newColor, newFormat;
-            newFormat = $(event.currentTarget).html().toLowerCase().replace("%", "p");
-            newColor = _this.getColor();
-            colorObject = tinycolor(newColor);
+            var newFormat   = $(event.currentTarget).html().toLowerCase().replace("%", "p"),
+                newColor    = self.getColor().toString(),
```

Comment:
> Ignoring case, I think the only time that ` toString()` returns something different from `getOriginalInput()` is between the time an change is made in inline color editor and `ColorEditor._commitColor()` is called, right? If so, then `getOriginalInput()` can always be used in unit tests.

That is correct. 

> I'm ok with that, but another solution would be to add a new method something like `ColorEditor.prototype.getColorString()` that calls `TinyColor.toString()` and then converts the case, if necessary.

If you recall, in the toggle-button implementation I actually had method by that name which did just that. However, I removed it in the refactoring. I can always restore it, but that may be better suited for a different PR.